test/integration/daemon/test_daemon.py

```diff
@@ -269,6 +270,7 @@ def test_daemon_http_requests_with_cfg(tt_cmd, tmpdir_with_cfg):
     status_rc, status_out = utils.run_command_and_get_output(status_cmd, cwd=tmpdir)
     assert status_rc == 0
     assert re.search(r"RUNNING. PID: \d+.", status_out)
+    time.sleep(5)
```

Comment:
Why do we need to add a timeout here and why is it so long (5 seconds)?

Reply:
Deleted